source-formatter.properties

```diff
@@ -1,3 +1,4 @@
 source.formatter.excludes=\
     src/main/resources/**,\
-    src/test/resources/**
\ No newline at end of file
+    src/test/resources/**.\
+    out/**
```

Comment:
This should go in .gitignore instead.  What is getting added to `out`?

Reply:
we already add it into .gitignore but sf tool still work on it.
when we run the code with coverage, it will be added.